backend/app/services/diagnosis_agent.py

```
@@ -0,0 +1,486 @@
+        if isinstance(weather_info, str):
+            weather_info = {}
+
+        # 날씨 필드명 통일 (랭플로우와 동일하게)
+        temp = str(weather_info.get("temperature", "데이터 없음"))
+        humi = str(weather_info.get("humidity", "데이터 없음"))
+        wind = str(weather_info.get("wind_speed", "데이터 없음"))
+        rain = str(weather_info.get("precipitation_prob", "데이터 없음"))
+        
+        try:
+            grouped_results = json.loads(state.get("pesticide_data", "[]"))
+        except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)